app/models/miq_provision_virt_workflow.rb

```diff
@@ -517,10 +517,9 @@ def allowed_templates(options = {})
     end
 
     rails_logger('allowed_templates', 0)
-    vms = []
+    vms = VmOrTemplate.all
```

Comment:
This is a great statement.

Instead of passing in `[]` to mean all, we're actually passing in `all` to mean all.

Think this is the only place where `[]` means all. So `filtered` and `search` can have the same meaning for `:targets => []`